### PR TITLE
Adds placeholders

### PR DESCRIPTION
**Pull request by v-bpastiu**
- adds placeholders for DOCR images and kube context for tilt

### CODE REVIEW

File: microservices-demo/Tiltfile-dev [hunk 1]
```diff
@@ -1,2 +1,2 @@
-allow_k8s_contexts('do-nyc1-starterkit-cluster-2')
+allow_k8s_contexts('<YOUR_KUBERNETES_CONTEXT>')
```
Comment (v-ctiutiu) [inline microservices-demo/Tiltfile-dev:1]: I would say:

```
allow_k8s_contexts('<YOUR_KUBERNETES_CONTEXT_HERE>')
```

File: microservices-demo/kustomize/base/cartservice.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ spec:
           privileged: false
           readOnlyRootFilesystem: true
         imagePullPolicy: Always
-        image: registry.digitalocean.com/microservices-demo/cartservice:1.0.0
+        image: <YOUR_DOCR_IMAGE>
```
Comment (v-ctiutiu) [inline microservices-demo/kustomize/base/cartservice.yaml:45]: I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/cartservice:1.0.0
```

File: microservices-demo/kustomize/base/checkoutservice.yaml [hunk 3]
```diff
@@ -41,7 +41,7 @@ spec:
             privileged: false
             readOnlyRootFilesystem: true
           imagePullPolicy: Always
-          image: registry.digitalocean.com/microservices-demo/checkoutservice:1.0.0
+          image: <YOUR_DOCR_IMAGE>
```
Comment (v-ctiutiu) [inline microservices-demo/kustomize/base/checkoutservice.yaml:44]: I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/checkoutservice:1.0.0
```

File: microservices-demo/kustomize/base/currencyservice.yaml [hunk 4]
```diff
@@ -42,7 +42,7 @@ spec:
           privileged: false
           readOnlyRootFilesystem: true
         imagePullPolicy: Always
-        image: registry.digitalocean.com/microservices-demo/currencyservice:1.0.0
+        image: <YOUR_DOCR_IMAGE>
```
Comment (v-ctiutiu) [inline microservices-demo/kustomize/base/currencyservice.yaml:45]: I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/currencyservice:1.0.0
```

File: microservices-demo/kustomize/base/emailservice.yaml [hunk 5]
```diff
@@ -42,7 +42,7 @@ spec:
           privileged: false
           readOnlyRootFilesystem: true
         imagePullPolicy: Always
-        image: registry.digitalocean.com/microservices-demo/emailservice:1.0.0
+        image: <YOUR_DOCR_IMAGE>
```
Comment (v-ctiutiu) [inline microservices-demo/kustomize/base/emailservice.yaml:45]: I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/emailservice:1.0.0
```

File: microservices-demo/kustomize/base/paymentservice.yaml [hunk 6]
```diff
@@ -42,7 +42,7 @@ spec:
           privileged: false
           readOnlyRootFilesystem: true
         imagePullPolicy: Always
-        image: registry.digitalocean.com/microservices-demo/paymentservice:1.0.0
+        image: <YOUR_DOCR_IMAGE>
```
Comment (v-ctiutiu) [inline microservices-demo/kustomize/base/paymentservice.yaml:45]: I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/paymentservice:1.0.0
```

File: microservices-demo/kustomize/base/productcatalogservice.yaml [hunk 7]
```diff
@@ -42,7 +42,7 @@ spec:
           privileged: false
           readOnlyRootFilesystem: true
         imagePullPolicy: Always
-        image: registry.digitalocean.com/microservices-demo/productcatalogservice:1.0.0
+        image: <YOUR_DOCR_IMAGE>
```
Comment (v-ctiutiu) [inline microservices-demo/kustomize/base/productcatalogservice.yaml:45]: I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/productcatalogservice:1.0.0
```

File: microservices-demo/kustomize/base/recommendationservice.yaml [hunk 8]
```diff
@@ -42,7 +42,7 @@ spec:
           privileged: false
           readOnlyRootFilesystem: true
         imagePullPolicy: Always
-        image: registry.digitalocean.com/microservices-demo/recommendationservice:1.0.0
+        image: <YOUR_DOCR_IMAGE>
```
Comment (v-ctiutiu) [inline microservices-demo/kustomize/base/recommendationservice.yaml:45]: I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/recommendationservice:1.0.0
```

File: microservices-demo/kustomize/base/shippingservice.yaml [hunk 9]
```diff
@@ -41,7 +41,7 @@ spec:
           privileged: false
           readOnlyRootFilesystem: true
         imagePullPolicy: Always
-        image: registry.digitalocean.com/microservices-demo/shippingservice:1.0.0
+        image: <YOUR_DOCR_IMAGE>
```
Comment (v-ctiutiu) [inline microservices-demo/kustomize/base/shippingservice.yaml:44]: I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/shippingservice:1.0.0
```

**Review by v-ctiutiu — CHANGES_REQUESTED** [review]

**Review by v-ctiutiu — APPROVED** [review]
Looks good to me now.

**Review by bikram20 — APPROVED** [review]

**Pull request merged**